pkg/ebpf/types_windows.go

```diff
@@ -0,0 +1,105 @@
+// +build windows
+
+package ebpf
+
+type FilterAddress struct {
+	addressFamily uint16
+	addr [16]byte
+	mask uint16
+}
+
+type FilterDirection uint32
+
+const (
+	DIRECTIONINBOUND FilterDirection = 0
+
+	// OUTGOING represents outbound connections from the host
+	DIRECTONOUTBOUND FilterDirection = 1
+)
+
+type FilterDefinition struct {
+	size uint32
+	addressFamily uint16
+	src FilterAddress
+	dst FilterAddress
+	sPort uint16
+	dPort uint16
+	protocol uint16
+	direction FilterDirection
+}
+
+type FilterPacketHeader struct {
+	size uint32
+	skippedSinceLast uint32
+	filterID uint64
+	direction uint32
+	pkt []byte
```

Comment:
this will almost certainly throw off the structure.
You want `pkt uintptr` here. We also may just remove that (if we also remove it from the driver) b/c I don't think it's actually useful.

Reply:
Dicussed elsewhere: this and other structures will likely live only in cgo.

Reply:
I'm resolving this since the packet structure has changed to a byte array.